src/js/index.js

```diff
@@ -85,3 +85,69 @@ $(document).ready(function(){
   }).resize();
 
 }); //jQuery
+
+//smooth scrolling
+$(function() {
```

Comment:
This should probably be reformatted to use 2 spaces per level of indentation, rather than tab characters.